akka-docs/rst/common/circuitbreaker.rst

```diff
@@ -45,8 +45,8 @@ What do they do?
 * In `Half-Open` state:
 	* The first call attempted is allowed through without failing fast
 	* All other calls fail-fast with an exception just as in `Open` state
-	* If the first call succeeds, the breaker is reset back to `Closed` state
-	* If the first call fails, the breaker is tripped again into the `Open` state for another full `resetTimeout`
+	* If the first call succeeds, the breaker is reset back to `Closed` state and the `resetTimeout` is reset
+	* If the first call fails, the breaker is tripped again into the `Open` state (as for exponential backoff circuit breaker, the `resetTimeout` is multiplied by the exponential backoff factor)
```

Comment:
Hmm. Isn't "exponential backoff" a very specific algorithm (not used here?) `1 / 2 * (2 ^ c - 1)` or something like that?